README.md

```diff
@@ -114,6 +115,15 @@ strategies:
            "pods": 50
 ```
 
+Policy should pass the following validation checks:
+* Only three types of resource are supported: `cpu`, `memory` and `pods`.
+* `thresholds` or `targetThresholds` can not be nil and they must configure exactly the same types of resources.
+* The valid range of the resource's percentage value is \[0, 100\]
+* Percentage value of `thresholds` can not be greater than `targetThresholds` for the same resource.
+
+You can configure any number of these three resources, if one resource is not configured, it will be set to default
```

Comment:
`If any of the resource types is not specified, all its thresholds default to 100% to avoid nodes going from underutilized to overutilized`.

Reply:
All these modified